scripts/inference.py

```diff
@@ -172,6 +218,7 @@ def infer(use_cache, do_sample):
         ids,
         max_new_tokens=100,
         use_cache=use_cache,
+        kv_cache_manager=kv_cache_manager,
```

Comment:
does this work? seems like the value is being used before it has been set.

Reply:
yes, this is what I have been using for testing, though I believe it should be set further above for simplicity